salt/client/ssh/shell.py

```diff
@@ -223,11 +225,18 @@ def _cmd_str(self, cmd, ssh='ssh'):
             opts = self._passwd_opts()
         if self.priv:
             opts = self._key_opts()
-        return "{0} {1} {2} {3} {4}".format(
+
+        ports = ''
+        if self.remote_port_forwards:
+            port_forwards = self.remote_port_forwards.split(',')
+            ports = ' '.join(map(lambda x: '-R {0}'.format(x), port_forwards))
```

Comment:
heh, personally I don't like them just because it can take some time for someone reading to code to grok what is going on.